extension/headerssetterextension/metadata.yaml

```
@@ -10,3 +10,5 @@ status:
   - sumo
   codeowners:
     active: [jpkrohling]
+tests:
+  config:
```

```suggestion
  config:

```
Nit, but we should probably end the file with a newline.